src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        Random random = new Random();
+        int index = random.nextInt(DiffColors.values().length);
+        DiffColors color = DiffColors.values()[index];
+        return color.toString();
```

Comment:
Use the `name()` method instead of `toString()` for getting the string representation of enum constants. This ensures that you always get the correct name of the enum constant, as `toString()` can be overridden and may not return the expected value.